### DISCUSSION TITLE
Prevent left clicking on selected page tab from cycling back and forth between previous active website tabs.

### DISCUSSION THREAD

**Discussion by AdSecIT** [mbnuqw/sidebery#1193] [Q&A]
Is it possible to stop left clicking on a selected website tab and have it cycle between the previous active website tab and the one that is selected?

For example if I click on a website tab on the tree while it is selected would like nothing to happen.

Thanks

--Edit Using latest version 5

**Comment by albino1** [comment]
Try disabling:

`Settings` -> `Mouse` -> `Tab Actions` -> `Activate previously active tab when clicking on the active tab (Tab flip)`

**Reply by AdSecIT** [reply]
That worked, thank you very much.
